crates/egui/src/containers/scroll_area.rs

```diff
@@ -90,6 +90,8 @@ pub struct ScrollArea {
     offset_y: Option<f32>,
     /// If false, we ignore scroll events.
     scrolling_enabled: bool,
+    /// If false we ignore mouse drag events.
+    drag_to_scroll_enabled: bool,
```

Comment:
When would you want to turn this off for just one ScrollArea an not another?
Wouldn't it make more sense to put this as global setting in `Style` instead?